test/unit/spec/narrative_core/upload/stagingAreaViewer-spec.js

```diff
@@ -155,30 +180,24 @@ define([
             ).toContain(fakeUser);
         });
 
-        it('Should show an error when a path does not exist', async () => {
+        it('Should show an error when file listing fails', async () => {
             const errorText = 'An error occurred while fetching your files';
-            jasmine.Ajax.stubRequest(/.*\/staging_service\/list\/foo?/).andReturn({
+            jasmine.Ajax.stubRequest(new RegExp(`${stagingServiceUrl}/list`)).andReturn({
```

Comment:
No longer call `/list/` on path changes, so stub out the whole function and re-render.